examples/physics/falling-shapes.html

```diff
@@ -134,7 +134,7 @@
         var boxTemplate = new pc.Entity();
         boxTemplate.addComponent("model", {
             type: "box",
-            castShadows: true
+            castShadows: true,
```

Comment:
I'd rather not hack a use case of the `offset` property into an existing engine example if it doesn't demonstrate something interesting. Maybe think of a new engine example that can show this feature off in a more interesting way?

Reply:
You've left a trailing comma here.

Reply:
Actually, if you've created a collision offset example, I think it's worth reverting this whole file.